util.py

```diff
@@ -279,19 +282,27 @@ def __new__(cls, degrees, coor_delta, tile_property):
         obj.map_property = tile_property
         return obj
 
+
+    def __add__(self, other):
+        return Direction((self.value + other.value) % 360)
+
+
     def get_new_direction(self, where_to):
         """
         Get new direction of given object.
 
-        Change attribute direction according to argument where_to, passed from TDB class DirectionOfRotation.
+        Change attribute direction according to argument where_to, passed from class Rotation.
         """
-        if where_to == "right":
-            return Direction((self.value + 90) % 360)
-        if where_to == "left":
-            return Direction((self.value + 270) % 360)
-        if where_to == "upside_down":
-            return Direction((self.value + 180) % 360)
+        return Direction(self + where_to)
+
 
+class Rotation(Enum):
+    """
+    Class describing the direction of the movement of the object-robot (dynamic).
+    """
+    LEFT = -90
+    RIGHT = 90
+    UP_DOWN = 180
```

Comment:
Nebyl by pro ↷ lepší název než nahoru-dolů?

Reply:
Doufám, že ano. "upside_down" je nepříjemně dlouhé, "180" je číslo, možná "revert" - ale může nebýt na první pohled jasné, co to znamená. Máš nějaký návrh?

Reply:
Ve hře je to U-Turn

Reply:
`U` je Určitě nejkratší!
Nebo “back”, “around”, “reverse”?

Reply:
u_turn it is.